test/e2e/storage/testsuites/snapshottable.go

```diff
@@ -451,11 +457,9 @@ func (s *snapshottableTestSuite) DefineTests(driver storageframework.TestDriver,
 					StopPod(cs, restoredPod)
 				})
 				framework.ExpectNoError(e2epod.WaitTimeoutForPodRunningInNamespace(cs, restoredPod.Name, restoredPod.Namespace, f.Timeouts.PodStartSlow))
-				if pattern.VolType != storageframework.GenericEphemeralVolume {
```

Comment:
this pattern check is no longer needed here?

Reply:
yeah not needed, the first lines of this test skip it if it's ephemeral so when we get here it's guaranteed that it's not ephemeral, there's another test that only runs for the ephemeral case